recipe/meta.yaml

```diff
@@ -13,16 +13,14 @@ source:
   {{ hash_type }}: {{ hash_val }}
 
 build:
-  number: 0
+  number: 1
   script: python setup.py install --single-version-externally-managed --record=record.txt
 
 
 requirements:
   build:
     - python
     - setuptools
-    - botocore >=1.3.0,<2.0.0
-    - futures >=2.2.0,<4.0.0  # [not py3k]
```

Comment:
These requirements are specified by s3transfer's setup.py
https://github.com/boto/s3transfer/blob/0.1.10/setup.py#L14
https://github.com/boto/s3transfer/blob/0.1.10/setup.py#L21

Are you sure they can be removed?